test/test_unary_ufuncs.py

```diff
@@ -1266,11 +1266,30 @@ def check_equal(t):
             self.assertEqual(actual, expected)
 
         range = (-10, 10)
+        t = torch.linspace(*range, 1, device=device, dtype=dtype)
+        check_equal(t)
 
-        t = torch.linspace(*range, int(1e4), device=device, dtype=dtype)
+        # Skip testing NaN, inf, -inf since they are tested in reference_numerics tests.
+        info = torch.finfo(dtype)
+        min, max, eps, tiny = info.min, info.max, info.eps, info.tiny
+        t = torch.tensor([min, max, eps, tiny], dtype=dtype, device=device)
+        check_equal(t)
+    
+    @dtypes(torch.float32, torch.float64)
+    @unittest.skipIf(not TEST_SCIPY, "SciPy not found")
+    def test_special_log_ndtr_vs_scipy(self, device, dtype):
```

Comment:
I think this test case is redundant with the above reference numerics tests? It's possibly those don't include min or tiny or eps or max explicitly, in which case I would add them to the extremal values test

Reply:
I tried adding (min, max, tiny, eps) to the extremal test cases, it's causing a lot more test cases to fail... Here are some failing error messages: 

Input tensor was 3.4028234663852886e+38, torch result is **inf**, and reference result is **3.575349975927421e+38.**

For extremely tiny/large numbers, we need to redefine the meaning of values_are_close. 

I agree these cases should be added, and it should be done in a separate PR. I will raise an issue to track this task.

Reply:
https://github.com/pytorch/pytorch/issues/74911 
issues created.